CONTRIBUTING.md

```diff
@@ -106,12 +106,12 @@ If you need to test that the image works with Cypress, you can follow these inst
 1. Install XQuartz: https://www.xquartz.org/
 2. Launch XQuartz. Under the XQuartz menu, select Settings
 3. Go to the security tab and ensure "Allow connections from network clients" is checked.
-4. From the XQuarts terminal, run `xhost + ${hostname}` to allow connections to the macOS host
-5. From the XQuarts terminal, set up a `HOSTNAME` env var `export HOSTNAME="host.docker.internal:0"`
-6. From the XQuarts terminal, run your Docker image like such:
-
+4. `xhost +$(ipconfig getifaddr en0)` to allow connections to the macOS host, this adds IP related to our en0 network interface to the access control list
+ - This step is vital for us to see the interactive UI for cypress 
```

Comment:
This line could probably be deleted, because the whole description is about how to get the interactive UI displayed.